pkg/kubectl/cmd/delete_test.go

```diff
@@ -99,17 +99,17 @@ func TestDeleteObjectByTuple(t *testing.T) {
 	}
 }
 
-func hasExpectedOrphanDependents(body io.ReadCloser, expectedOrphanDependents *bool) bool {
-	if body == nil || expectedOrphanDependents == nil {
-		return body == nil && expectedOrphanDependents == nil
+func hasExpectedPropagationPolicy(body io.ReadCloser, policy *metav1.DeletionPropagation) bool {
+	if body == nil || policy == nil {
+		return body == nil && policy == nil
 	}
 	var parsedBody metav1.DeleteOptions
 	rawBody, _ := ioutil.ReadAll(body)
 	json.Unmarshal(rawBody, &parsedBody)
```

Comment:
No error handling on these two lines. Maybe fix it as a separate commit while at it?

Reply:
Or maybe its ok...

Reply:
I think it's fine - it's part of a unit test which will break if anything is wrong with the body.